images/stackstorm-1ppc/nginx_st2.conf

```diff
@@ -0,0 +1,151 @@
+#
+# nginx configuration to expose st2 webui, redirect HTTP->HTTPS,
+# provide SSL termination, and reverse-proxy st2api and st2auth API endpoint.
+# To enable:
+#    cp ${LOCATION}/st2.conf /etc/nginx/sites-available
+#    ln -l /etc/nginx/sites-available/st2.conf /etc/nginx/sites-enabled/st2.conf
+# see https://docs.stackstorm.com/install.html for details
+
+server {
+  listen *:80 default_server;
+
+  add_header Front-End-Https on;
+  add_header X-Content-Type-Options nosniff;
+
+  if ($ssl_protocol = "") {
+       return 301 https://$host$request_uri;
+  }
+
+  index  index.html;
+
+  access_log /var/log/nginx/st2webui.access.log combined;
+  error_log /var/log/nginx/st2webui.error.log;
+}
+
+server {
+  listen       *:443 ssl;
+
+  ssl on;
+
+  ssl_certificate           /etc/ssl/st2/st2.crt;
+  ssl_certificate_key       /etc/ssl/st2/st2.key;
+  ssl_session_cache         shared:SSL:10m;
+  ssl_session_timeout       5m;
+  ssl_protocols             TLSv1 TLSv1.1 TLSv1.2;
+  ssl_ciphers               EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH:ECDHE-RSA-AES128-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA128:DHE-RSA-AES128-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES128-GCM-SHA128:ECDHE-RSA-AES128-SHA384:ECDHE-RSA-AES128-SHA128:ECDHE-RSA-AES128-SHA:ECDHE-RSA-AES128-SHA:DHE-RSA-AES128-SHA128:DHE-RSA-AES128-SHA128:DHE-RSA-AES128-SHA:DHE-RSA-AES128-SHA:AES128-GCM-SHA384:AES128-GCM-SHA128:AES128-SHA128:AES128-SHA128:AES128-SHA:AES128-SHA:HIGH:!aNULL:!eNULL:!EXPORT:!DES:!MD5:!PSK:!RC4;
```

Comment:
This nginx_st2.conf is based on `/etc/nginx/conf.d/st2.conf` in `stackstorm:stackstorm` image, and the modification is made as minimal as possible. I haven't touch those parameters, so if it's necessary then it should be fixed in upstream package I guess?